index.js

```diff
@@ -0,0 +1,64 @@
+const dotenv = require('dotenv')
+const { Bot } = require('grammy');
+
+dotenv.config();
+
+const bot = new Bot(process.env.BOT_TOKEN);
+
+bot.use((ctx, next) => {
+    ctx.state = {};
+    return next();
+})
+
+const middlewareMatchToSplit = function (ctx, next){
```

Comment:
`function (ctx, next){`
No space between ) and {

Reply:
Done

Reply:
Done